c/src/core/codec.c

```diff
@@ -657,7 +657,46 @@ int pn_data_vfill(pn_data_t *data, const char *fmt, va_list ap)
         }
       }
       break;
-    default:
+     case 'M': {                /* multiple=true field value */
+       /* Encode a pn_data_t * as a normalized multiple="true" field.
+          See file:///home/aconway/specs/amqp-core-complete-v1.0.pdf
```

Comment:
What is 'multiple="true" field'? There is nothing called this in the standard is there? The ="true' naming seems to indicate boolean fields, but this is not the case afaict - and this really applies to a list of things that can also be empty in which case we just use null instead of the empty list.

Also url to local file system is not very helpful!